org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/ILanguageServerInfoProvider.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.jdt.ls.debug.adapter;
+
+public interface ILanguageServerInfoProvider {
+    String getFullyQualifiedName(String sourceFilePath, int line, int column);
```

Comment:
we don't support column breakpoint. so here just need two parameters `sourceFilePath` and `line`